config/prometheus/prometheus.yml

```
@@ -0,0 +1,115 @@
+    environment: 'development'
+
+# Rule files specifies a list of files from which rules are read
+rule_files:
+  - "/etc/prometheus/rules/*.yml"
+
+# Alerting specifies settings related to the Alertmanager
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)